styles/globals.css

```diff
@@ -54,4 +54,11 @@ p, a, li, span, input, textarea, button {
 
 iframe{
   background-color: #fff;
+}
+
+.pad{
```

Comment:
no need to add custom classes, we can do this behavior using `tailwind` CSS classes.